manager/manager.go

```diff
@@ -218,7 +221,7 @@ func New(config *Config) (*Manager, error) {
 	m := &Manager{
 		config:      config,
 		listeners:   listeners,
-		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig),
+		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig, caReconciliationRetryInterval),
```

Comment:
Why not use the same trick you used for the flaky tests, and have a test server that allows you to change the param?